docs/secure-guardrails/custom-guardrails-rules.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
This doc was copied directly from the `s-santillan` fork except for 1 change (I will mark).